install.sh

```diff
@@ -106,6 +123,7 @@ sudo -u ${USER} bash <<EOF
     pip3 --no-cache-dir install paho-mqtt
     pip3 --no-cache-dir install toml
 	pip3 --no-cache-dir install pyyaml
+	${pipNumpy}
```

Comment:
Maybe there is a more elegant way to do this.
I'm not that comfortable using bash/python/pip. Was the simplest way I found.